src/visualizations/App.fs

```diff
@@ -6,7 +6,16 @@ open Feliz
 
 open Types
 
-let init (query : obj) (visualization : string option) =
+let getTranslation lang = 
+    match lang with
+    | Slovenian -> 
+        { MetricsComparison = "Stanje COVID-19 v Sloveniji"
+          ConfirmedCases = "Potrjeni primeri" }
+    | English -> 
+        { MetricsComparison = "COVID-19 situation in Slovenia"
+          ConfirmedCases = "Confirmed cases" }
+
```

Comment:
Predvidevam, da je tole še potrebno predelave, da se bodo uproabljali isti prevodi iz
https://github.com/sledilnik/website/blob/ebc93d419a10d23bf46d4be222b7d98ab24ca889/src/locales/sl.json#L31-L68
in
https://github.com/sledilnik/website/blob/ebc93d419a10d23bf46d4be222b7d98ab24ca889/src/locales/en.json#L30-L55

Reply:
@joahim je dodal nov način nalaganja prevodov v F#, tako da je to zdaj rešeno.